Amy Lloyd is interested in leasing a new Honda and has contacted three
automobile dealers for pricing information. Each dealer offered Amy a closed-end
36-month lease with no down payment due at the time of signing. Each lease includes a
monthly charge and a mileage allowance. Additional miles receive a surcharge on a per-
mile basis. The monthly lease cost, the mileage allowance, and the cost for additional
miles are as follows:


<table>
    <tr>
        <th>Dealer</th>
        <th>Monthly Cost</th>
        <th>Mileage Allowance</th>
        <th>Cost per Additional Mile</th>
    </tr>
    <tr>
        <td>Hepburn Honda</td>
        <td>$299</td>
        <td>36,000</td>
        <td>$0.15</td>
    </tr>
    <tr>
        <td>Midtown Motors</td>
        <td>$310</td>
        <td>45,000</td>
        <td>$0.20</td>
    </tr>
    <tr>
        <td>Hopkins Automotive</td>
        <td>$325</td>
        <td>54,000</td>
        <td>$0.15</td>
    </tr>
</table>


Amy decided to choose the lease option that will minimize her total 36-month cost. The difficulty is that Amy is not sure how many miles she will drive over the next three years. For purposes of this decision, she believes it is reasonable to assume that she will drive 12,000 miles per year, 15,000 miles per year, or 18,000 miles per year. With this assumption Amy estimated her total costs for the three lease options. For example, she figures that the Hepburn Honda lease will cost her <br> <br>
$36($299) + $0.15(36,000 - 36,000) = $10,764$ if she drives 12,000 miles per year, <br>
$36($299) + $0.15(45,000 - 36,000) = $12,114$ if she drives 15,000 miles per year, <br>
$36($299) + $0.15(54,000 - 36,000) = $13,464$ if she drives 18,000 miles per year. <br>


a. What is the decision, and what is the chance event? <br>
b. Construct a payoff table for Amy’s problem. <br>
c. If Amy has no idea which of the three mileage assumptions is most appropriate, what is the recommended decision (leasing option) using the optimistic, conservative, and minimax regret approaches? <br>
d. Suppose that the probabilities that Amy drives 12,000, 15,000, and 18,000 miles per year are 0.5, 0.4, and 0.1, respectively. What option should Amy choose using the expected value approach? <br>
e. Develop a risk profile for the decision selected in part (d). What is the most likely cost, and what is its probability?<br>
f. Suppose that, after further consideration, Amy concludes that the probabilities
that she will drive 12,000, 15,000, and 18,000 miles per year are 0.3, 0.4, and 0.3,
respectively. What decision should Amy make using the expected value approach?



In [8]:
##### B ######
print((36 * 310) + (0.2 * (max(0, ((3 * 12_000) - 45_000)))))
print((36 * 310) + (0.2 * (max(0, ((3 * 15_000) - 45_000)))))
print((36 * 310) + (0.2 * (max(0, ((3 * 18_000) - 45_000)))))
# ---------------
print((36 * 325) + (0.15 * (max(0, ((3 * 12_000) - 54_000)))))
print((36 * 325) + (0.15 * (max(0, ((3 * 15_000) - 54_000)))))
print((36 * 325) + (0.15 * (max(0, ((3 * 18_000) - 54_000)))))

11160.0
11160.0
12960.0
11700.0
11700.0
11700.0


#### Solution:
a. Chance event is How many miles Amy will drive. <br>
______
b. 
<table>
    <tr>
        <th>Dealer</th>
        <th>Cost of 12000 miles</th>
        <th>Cost of 15000 miles</th>
        <th>Cost of 18000 miles</th>
    </tr>
    <tr>
        <td>Hepburn Honda</td>
        <td>10,764</td>
        <td>12,114</td>
        <td>13,464</td>
    </tr>
    <tr>
        <td>Midtown Motors</td>
        <td>11,160</td>
        <td>11,160</td>
        <td>12,960</td>
    </tr>
    <tr>
        <td>Hopkins Automotive</td>
        <td>11,700</td>
        <td>11,700</td>
        <td>11,700</td>
    </tr>
</table>

_____________
c.<br>
Optimistic (12,000 miles): Hepburn Honda - 10,764 <br>
Conservative (18,000 miles): Hopkins Automotive - 11,700 <br>
Minimax: 

<table>
    <tr>
        <th>Dealer</th>
        <th>Cost of 12000 miles</th>
        <th>Cost of 15000 miles</th>
        <th>Cost of 18000 miles</th>
    </tr>
    <tr>
        <td>Hepburn Honda</td>
        <td>0</td>
        <td>954</td>
        <td>1260</td>
    </tr>
    <tr>
        <td>Midtown Motors</td>
        <td>396</td>
        <td>0</td>
        <td>1764</td>
    </tr>
    <tr>
        <td>Hopkins Automotive</td>
        <td>936</td>
        <td>540</td>
        <td>0</td>
    </tr>
</table>

Meaning Hopkins Automotive is the best option with 1476.

_______________
d. <br>
Hupburn = $0.5(10746) + 0.4(12114) + 0.1(13464) = 11565$ <br>
Midtown = $0.5(11160) + 0.4(11160) + 0.1(12960) = 11340$ <br>
Hopkins = $0.5(11700) + 0.4(11700) + 0.1(11700) = 11700$ <br>

meaning Midtown motors would be the best choice

____________
e. <br><br>
<img src = "ImagesForSolutions/RiskCarLease.png">

11,160 would be the likely with 50%.


f. Decide on something Amy